thirdeye/thirdeye-pinot/src/main/resources/assets/js/lib/anomalies.js

```diff
@@ -13,7 +13,14 @@ function getAnomalies(tab) {
         var compareMode = hash.fnCompareWeeks ?  hash.fnCompareWeeks : 1;
         var baselineStart = moment(parseInt(hash.currentStart)).subtract(compareMode, 'week');
         var baselineEnd = moment(parseInt(hash.currentEnd)).subtract(compareMode, 'week');
-        var aggTimeGranularity = (window.datasetConfig.dataGranularity) ? window.datasetConfig.dataGranularity : "HOURS";
+        var aggTimeGranularity;
```

Comment:
Lets limit the magic numbers as less a possible. extract unit val to a var and then multiply numDays into it, will make the code more readable.

var numMillisInDay = 86400000;
var diff = parseInt(hash.currentEnd) - parseInt(hash.currentStart);
if (diff > 2 * numMillisInDay) {

} else if ...